compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -638,33 +636,38 @@ createProjectPackageDb opts fps = do
         -> Options
         -> FilePath
         -> FilePath
+        -> FilePath
         -> String
         -> String
         -> String
         -> Maybe String
         -> [String]
         -> IO ()
-    generateAndInstallIfaceFiles dalf src opts dbPathAbs projectPackageDatabaseAbs unitIdStr pkgIdStr pkgName mbPkgVersion deps = do
+    generateAndInstallIfaceFiles dalf src opts workDir dbPath projectPackageDatabase unitIdStr pkgIdStr pkgName mbPkgVersion deps = do
         loggerH <- getLogger opts "generate interface files"
-        mapM_ writeSrc src
+        let src' = [ (toNormalizedFilePath $ workDir </> fromNormalizedFilePath nfp, str) | (nfp, str) <- src]
+        mapM_ writeSrc src'
         opts' <-
             mkOptions $
             opts
-                { optWriteInterface = True
-                , optPackageDbs = projectPackageDatabaseAbs : optPackageDbs opts
-                , optIfaceDir = Just "./"
+                { optWriteInterface = False
```

Comment:
Why did this change?

Reply:
We write the interfaces with `writeIfaceAndHie` in the next step. This way we don't do it two times.

Reply:
So this was redundant before this PR as well? Makes sense in that case, thanks for the explanation!

Reply:
yes, this was redundant before.